java/test/jmri/implementation/configurexml/load/SignalHeadTestExamples.xml

```diff
@@ -252,5 +252,5 @@
       <filename />
     </operation>
   </filehistory>
-  <!--Written by JMRI version 4.13.5ish+jake+20181113T1609Z+Re7a9e1ae07 on Tue Nov 13 08:18:36 PST 2018-->
+  <!--Written by JMRI version 4.15.7ish+jake+20181113T1609Z+Re7a9e1ae07 on Tue Nov 13 08:18:36 PST 2018-->
```

Comment:
Might be better not to change the "written by", as it helps one understand how old the file actually is

Reply:
Reverted to 4.13.5